.npmrc

```
@@ -1 +1 @@
-auto-install-peers = false
+auto-install-peers = false
```

```suggestion
auto-install-peers = false

```
这个文件需要末尾行，否则 CI 会写坏文件：
https://github.com/kaiyuanshe/kaiyuanshe.github.io/blob/94d3427e0544b5970190de85520d4546862a7c05/.github/workflows/deploy-production.yml#L31-L35